### PR TITLE
feat: ignoring missing plugins when ASDF_DIRENV_IGNORE_MISSING_PLUGINS=1

### LINKED ISSUE #162: Question: Is `use asdf` supposed to fail on missing plugins?
Given the following `.tool-versions`:

```
ruby 3.1.2
nodejs 16.13.0
```

and the following `envrc`:

```
use asdf
```

If I don't have the nodejs plugin, the tool will report `direnv: asdf plugin not installed: nodejs` and exit.  In result, the path to Ruby will be not loaded. Meanwhile, pure asdf will ignore any entries in .tool-versions for which you do not have the plugins and just work for the ones you do.

Is it supposed to work like this?

### PR DESCRIPTION
**Pull request by robsonpeixoto**
closes #162

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -82,4 +82,4 @@ jobs:
         run: brew install shfmt
 
       - name: Run shfmt
-        run: make fmt-check
```
Comment (jfly) [inline .github/workflows/workflow.yml:85]: Did this have to get renamed? I'm not opposed to the rename, but can we put it in a different PR just to keep this one small?

File: lib/tools-environment-lib.bash [hunk 2]
```diff
@@ -265,14 +265,20 @@ _plugin_env_bash() {
...
 
   plugin_path=$(get_plugin_path "$plugin")
   if [ ! -d "$plugin_path" ]; then
-    log_error "asdf plugin not installed: $plugin"
-    exit 1
+    if [ "$ignore_missing_plugins" -eq "0" ]; then
+      log_error "asdf plugin not installed: $plugin"
+      exit 1
+    else
+      log_error "ignoring not installed plugin: $plugin"
```
Comment (jfly) [inline lib/tools-environment-lib.bash:279]: Do we have a `log_warning` we could use here instead?

**Review by jfly — COMMENTED** [review]
I like this. A couple of thoughts:

- can you try adding a unittest of this?
- can you add a new section our README for `Configuration`? We haven't had any configuration options before, so now we have to figure out how to document stuff.

File: .github/workflows/workflow.yml (hunk 1 above)
Comment (robsonpeixoto) [reply .github/workflows/workflow.yml:85]: Done: https://github.com/asdf-community/asdf-direnv/pull/173/files

**Review by jfly — COMMENTED** [review]
Code and docs look good! Can you add a unit test?

**Comment by robsonpeixoto** [comment]
The PR is ready to review @jfly

**Review by jfly — APPROVED** [review]
LGTM! Thanks, @robsonpeixoto!
